domains/k93msy.json

```diff
@@ -0,0 +1,12 @@
+
+{
+  "description": "K93MSY Web",
+  "repo": "https://github.com/K93MSY/K93MSY.github.io",
+  "owner": {
+    "username": "K93MSY",
+    "email": "kuritama0903@gmail.com"
+  },
+  "record": {
+    "CNAME": "K93MSY.github.io"
```

Comment:
this is linked to other website already.

Reply:
fixed